src/datanode/src/region_server.rs

```diff
@@ -201,7 +201,8 @@ impl RegionServerInner {
             | RegionRequest::Delete(_)
             | RegionRequest::Alter(_)
             | RegionRequest::Flush(_)
-            | RegionRequest::Compact(_) => RegionChange::None,
+            | RegionRequest::Compact(_)
+            | RegionRequest::Truncate(_) => RegionChange::None,
```

Comment:
Remember to update the [proto](https://github.com/GreptimeTeam/greptime-proto) as well

Reply:
Thx, I will update the proto in the next PR.